### ISSUE TITLE
Project-1

### ISSUE THREAD

**Issue by Hodo-** [Hodo-/dsn1674-project-1#1]
@thomasjbradley I was working on the project 1 and it was going well and all of the sudden everything just stopped working; like, for example, I would put text-align in the css for the cite tag and nothing worked. Nothing seemed to respond to anything I write in css. I wonder why this was happening and I didn't have this issue in the last few assignments.

**Comment by thomasjbradley** [comment]
Have you validated your CSS to make sure there are no missing semicolons or curly brackets?

**Comment by Hodo-** [comment]
Yeah, nothing out of ordinary. What should I do?

**Comment by Hodo-** [comment]
@thomasjbradley Yeah, nothing out of ordinary. What should I do?

**Comment by thomasjbradley** [comment]
Have you tried moving your `cite` tag to inside the `blockquote`?

**Comment by Hodo-** [comment]
@thomasjbradley Thank you so much! I have completed the assignment! Have a nice evening!

**Comment by Hodo-** [comment]
@thomasjbradley Since the project-1 is completed, I was going over the criticism you gave me on Assignment 2. I tried to put the cite tag on assignment 3 and 4 and try to style cite in css but it didn't work at all, even with "class". So, I used the p tag instead.

What is the personal critique? What am I supposed to do?

**Comment by Hodo-** [comment]
@thomasjbradley I forgot to mention one thing. I've made the mistake by uploading the assignment on canvas two times when I could just press refresh on the github app to save the changes. I was new to github at that time. I uploaded the link of the assignment 3 again today because I revised the title to take out the "revised" part. The first time I handed it in was on time as it was shown on the canvas.
